packages/nextra/src/server/compile.ts

```diff
@@ -289,6 +290,7 @@ export async function compileMdx(
                   ...rehypePrettyCodeOptions
                 }
               ] as any,
+              rehypeTwoslashPopup,
```

Comment:
similar to https://github.com/shuding/nextra/pull/3533/files#r1815445926 suggestion, to avoid warnings which comes from `typescript/vfs` we need to add this plugin in loader.ts file

Reply:
I had similar problem already which I fixed in https://github.com/shuding/nextra/pull/3349/files PR